### PR TITLE
Add better copy on the error pages

### PR DESCRIPTION
**Pull request by tavy88**
### Jira link

HOTT-<TODO>

### What?

I have added/removed/altered:

- [x] Add better copy on the error pages
- [x] Also return 404 for the root path as we don't have a page for that

### Why?

I am doing this because:

- Users should be able to navigate their way around a 500 or similar error and keep being able to use the service

### CODE REVIEW

File: app/views/errors/internal_server_error.html.erb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <main role="main" class="govuk-main-wrapper" id="main-content">
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">We're sorry, but something went wrong.</h1>
-      <p class="govuk-body">If you are the application owner check the logs for more information.</p>
+      <h1 class="govuk-heading-xl">Sorry, there is a problem with the service</h1>
+      <p class="govuk-body">We could not proceed with calculating your duty.</p>
+      <p class="govuk-body">Click here to <%= link_to('start again', import_date_path) %>.</p>
```
Comment (willfish) [inline app/views/errors/internal_server_error.html.erb:6]: Should we pass the query params to the import_date_path if they're available?

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  root to: proc { [404, {}, ['Not found.']] }
```
Comment (willfish) [inline config/routes.rb:2]: Should the mimetype of this be html?

**Review by willfish — APPROVED** [review]

File: app/views/errors/internal_server_error.html.erb (hunk 1 above)
Comment (tavy88) [reply app/views/errors/internal_server_error.html.erb:6]: Yeah, interesting point. They would already have it on the session though?

File: config/routes.rb (hunk 2 above)
Comment (tavy88) [reply config/routes.rb:2]: Does it matter that much?

**Pull request merged**